ext/RMagick/rmmain.c

```diff
@@ -1295,12 +1295,10 @@ Init_RMagick2(void)
     DEF_ENUM(PixelInterpolateMethod)
         ENUMERATOR(UndefinedInterpolatePixel)
         ENUMERATOR(AverageInterpolatePixel)
-        ENUMERATOR(BicubicInterpolatePixel)
         ENUMERATOR(BilinearInterpolatePixel)
-        ENUMERATOR(FilterInterpolatePixel)
         ENUMERATOR(IntegerInterpolatePixel)
         ENUMERATOR(MeshInterpolatePixel)
-        ENUMERATOR(NearestNeighborInterpolatePixel)
+        ENUMERATORV(NearestInterpolatePixel, NearestNeighborInterpolatePixel)
```

Comment:
So are we deprecating `NearestNeighborInterpolatePixel`, or are we going to keep them aliased for a while?

Reply:
Not sure what you are asking. I am on Skype now if you want to chat.

Reply:
Let's talk tomorrow if you're available. I'm pretty busy today.

Reply:
That will work for me.